source/Camera/Basic/CaptureFromFile.m

```diff
@@ -0,0 +1,19 @@
+try
+    app = zividApplication;
+
+    zdfFile = strcat(char(Zivid.NET.Environment.DataPath), '/MiscObjects.zdf');    
+    disp(['Initializing camera emulation using file: ', zdfFile]);
+    camera = app.CreateFileCamera(zdfFile);
+
+    disp('Capture a frame');
+    frame = camera.Capture();
+
+    resultFile = 'result.zdf';
+    disp(['Saving frame to file: ', resultFile]);    
+	frame.Save(resultFile);
```

Comment:
\t extra

Reply:
Space after colon? Deleted.

Reply:
No, this was correct before, but frame.Save(resultFile); have wrong tab???

Reply:
Done